specs/latest/1.0/index.html

```diff
@@ -395,6 +395,54 @@ <h3><a name="THE_DRAWING_BUFFER">The Drawing Buffer</a></h3>
         </p>
     </div>
 
+    <dl class="methods">
+        <dt class="idl-code">undefined backbufferStorage(GLenum sizedFormat, unsigned long width, unsigned long height);
+        <dd>
+            Respecify the size and format of the backbuffer,
+            similar to <code>renderbufferStorage</code> as used on Renderbuffers.
+
+            Clearing behavior is equivalent to setting <code>HTMLCanvasElement.width</code> and <code>HTMLCanvasElement.height</code>, followed by changing the backbuffer format.
```

Comment:
The spec talks about clearing behavior here.

I presume this means that after the developer calls `renderbufferStorage`, subsequently querying `HTMLCanvasElement.width`, `HTMLCanvasElement.height`, `drawingBufferWidth` and `drawingBufferHeight` will return the values passed to `renderBufferstorage'?

Reply:
Yes, modulo the same limitations the browser already enforces. Recall that `drawingBufferWidth` and `drawingBufferHeight` may be clamped due to platform limitations. The intent here is to make this new API behave exactly the same as resizing the `HTMLCanvasElement` or `OffscreenCanvas`, to avoid adding yet another behavior to the web platform.